js/go112_wasm.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 The GopherWasm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build wasm
+// +build go1.12
+
+package js
+
+import (
+	"syscall/js"
+)
+
+type Func = js.Func
+
+func FuncOf(fn func(this Value, args []Value) interface{}) Func {
+	return js.FuncOf(fn)
+}
+
+// Callback is for backward compatibility. Use Func instead.
+type Callback = js.Func
+
+// EventCallbackFlag is for backward compatibility.
+type EventCallbackFlag int
+
+const (
+	PreventDefault EventCallbackFlag = 1 << iota
+	StopPropagation
+	StopImmediatePropagation
+)
+
+// NewCallback is for backward compatibility. Use FuncOf instead.
+func NewCallback(fn func([]Value)) Callback {
+	return FuncOf(func(this Value, args []Value) interface{} {
+		go func() {
+			fn(args)
+		}()
+		return nil
+	})
+}
+
+// NewEventCallback is for backward compatibility. Use FuncOf instead.
+func NewEventCallback(flags EventCallbackFlag, fn func(event Value)) Callback {
+	return FuncOf(func(this Value, args []Value) interface{} {
+		e := js.Undefined()
+		if len(args) > 0 {
```

Comment:
This check makes the behavior different to Go 1.11:
https://github.com/golang/go/blob/release-branch.go1.11/misc/wasm/wasm_exec.js#L398

Reply:
`_makeEventCallbackHelper` expects that the `event` argument is given, otherwise it crashes. It is more consistent to replicate this behavior.

Reply:
Ah, you mean that we should let it crash? Sure.

Reply:
Done.